test/Common/ApplicationFactories/IdentityApplicationFactory.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Bit.Core.Enums;
+using Bit.Core.Models.Api.Request.Accounts;
+using Bit.Core.Utilities;
+using Bit.Identity;
+using Bit.Test.Common.Helpers;
+using Microsoft.AspNetCore.Http;
+
+namespace Bit.Test.Common.ApplicationFactories
+{
+    public class IdentityApplicationFactory : WebApplicationFactoryBase<Startup>
```

Comment:
Bit of a thought, are we supposed to have this in `Common`? Will it be used by other projects than `Identity.IntegrationTest`?

Reply:
Exactly, I think at the least the IdentityApplicationFactory will be used to test other projects factories and for that reason it is going to have typed versions of common endpoints. I wasn't counting out the possibility that other projects would have similar calls that would be needed to test other functionality and that's why I think those should live in a shared project.